Packs/Devo/Integrations/Devo_v2/Devo_v2.yml

```diff
@@ -54,24 +53,26 @@ configuration:
     name: fetch_incidents_filters
     type: 12
     required: false
-  - display: Deduplication parameters JSON if required
+  - display: (deprecated) Deduplication parameters JSON if required
```

Comment:
The validation failed on 


> [ERROR]: Packs/Devo/Integrations/Devo_v2/Devo_v2.yml: [IN135] - The integration display names: ['(deprecated) Deduplication parameters JSON if required'] are invalid, Integration parameters display name should be capitalized and spaced using whitespaces and not underscores ( _ ).

you can delete the "(deprecated)"

Reply:
Regarding the 

> [ERROR]: Packs/Devo/Integrations/Devo_v2/Devo_v2.yml: [IN124] - Parameter: "fetch_incidents_deduplication" can't be hidden in all marketplaces. Please either remove the `hidden` attribute, or replace its value with a list of marketplace names, where you wish it to be hidden.

You can add the validation code (IN124) to the pack_ignore file.
You can see docs and example [here](https://xsoar.pan.dev/docs/packs/packs-format#pack-ignore)